Kickstarter-iOS/Views/Controllers/LiveVideoViewController.swift

```diff
@@ -231,3 +242,19 @@ extension LiveVideoViewController: OTSessionDelegate {
     self.viewModel.inputs.sessionDidFailWithError(error: error)
   }
 }
+
+extension LiveVideoViewController: OTSubscriberDelegate {
+  public func subscriberDidConnect(toStream subscriber: OTSubscriberKit) {}
+
+  public func subscriber(_ subscriber: OTSubscriberKit, didFailWithError error: OTError) {}
```

Comment:
This needs to be implemented for protocol conformance but we've not cared about this error'ing yet. I want to consider its effect and implement it in a separate PR.